### ISSUE TITLE
issue with [Bedrockify](https://www.curseforge.com/minecraft/mc-mods/bedrockify) when quitting the game while a world is saving

### ISSUE THREAD

**Issue by Linusintro** [KingContaria/FastQuit#33]
[with bedrockify.zip](https://github.com/KingContaria/FastQuit/files/12500968/with.bedrockify.zip)
There is an issue with [Bedrockify](https://www.curseforge.com/minecraft/mc-mods/bedrockify) when quitting the game while a world is saving. It's no malfunction or a crash but a graphical issue. It's this moving thing from the multiplayer screen and I would suggest to just add a toggle to the config file. I've attached an image in a zip file.

**Comment by KingContaria** [comment]
i dont think it needs a toggle i'll just add a compat check if bedrockify is present and has that option enabled
i'll put 3.0.0 out of beta first tho, have been procrastinating on that for way too long now

**Comment by Linusintro** [comment]
Thanks!
and please for 1.19 too.

**Comment by contariaa** [comment]
This issue was resolved by removing the animation entirely when the Minecraft screen background was changed, I don't currently plan to update FastQuit for anything but the newest version so I'll close this issue
